dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace CopyObject
+{
+    public class CopyObject
+    {
+        // Specify your bucket region (an example region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the names for the Simple Storage Service objects
+        private const string SOURCE_BUCKET_NAME = "*** provide the name of the bucket with source object ***";
```

Comment:
You should provide actual, example or "dummy" values for all constants. This is especially helpful even for certain types of strings where you need to specify Amazon Resource Names (ARNs), where they are all kinds of : and :: and / and etc.

Reply:
I try to use names like: aws-docs-example-bucket-name

Don't forget that:

Bucket names must be between 3 and 63 characters long. Bucket names can consist only of lowercase letters, numbers, dots (.), and hyphens (-). Bucket names must begin and end with a letter or number.

Reply:
Note that for Amazon S3, the only approved "dummy" bucket names for us currently are doc-example-bucket, doc-example-bucket1, and doc-example-bucket2. Other "dummy" bucket names could actually exist, and we don't have any ownership of them.

Reply:
Thanks -- changes made throughout.